### DISCUSSION TITLE
reveal.js 需要梯子才能打开吗？

### DISCUSSION THREAD

**Discussion by heroxbd** [physics-data/faq#60] [Q&A] [answered]
有没有内部的 css 镜像？

**Comment by Harry-Chen** [comment, marked as answer]
Try https://cdn.jsdelivr.net/npm/reveal.js/

**Comment by YiHui-Liu** [comment]
try：https://cdn.bootcdn.net/ajax/libs/reveal.js/4.0.2/js/reveal.js

**Reply by Harry-Chen** [reply]
不推荐 bootcdn，并不稳定
